### PR TITLE
feat(Mint Assets): Update minted tokens view with assets

### LINKED ISSUE #10625: [Community Mint Assets] Update minted tokens view with assets
### Description

[Design](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29384-569202&t=gvDNrxNvn9mQnFYl-0):

- Add list of assets in `CommunityMintedTokensView`:
Screenshot 2023-05-10 at 17 06 53

Consider all asset state possibilities (minting, failure).

- Add empty box for assets / collectibles:
Screenshot 2023-05-10 at 17 54 44

**NOTE:** Look at `CommunityMintTokensSettingsPanel` and `CommunityMintedTokensView`

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #10625 and #10765

### What does the PR do

- It adds assets list UI in minted tokens view.
- It adds flow to open token view depending on type (asset or collectible).
- It updates minting flow to use `contractUniqueKey` instead of `index`.
- It refactors `SettingsPageLayout`.
- It adds `retry mint` and `delete` options when deploy process fails.
- It renames `RemotelyDestructAlertPopup` to generic name `AlertPopup`.
- It updates `storybook` accordingly.

**NOTE:** Only UI. Assets backend management is not ready, neither delete or retry minting.

### Affected areas

Community Settings / Mint Tokens

### Screenshot of functionality

https://github.com/status-im/status-desktop/assets/97019400/a6409554-6517-46d1-8062-eb1a948a6595

https://github.com/status-im/status-desktop/assets/97019400/696a671a-3a69-4012-a06d-4f75e550dc49

### CODE REVIEW

File: ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml [hunk 1]
```diff
@@ -421,10 +438,16 @@ SettingsPageLayout {
...
-            RemotelyDestructAlertPopup {
+            AlertPopup {
                 id: alertPopup
 
-                onRemotelyDestructClicked: {
+                property int tokenCount
+
+                title: qsTr("Remotely destruct %n token(s)", "", tokenCount)
+                acceptBtnText: qsTr("Remotely destruct")
+                alertText: qsTr("Continuing will destroy tokens held by members and revoke any perissions they given. To undo you will have to issue them new tokens.")
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml:448]:
```suggestion
                alertText: qsTr("Continuing will destroy tokens held by members and revoke any permissions they are given. To undo you will have to issue them new tokens.")
```

**Review by caybro — APPROVED** [review]
Looks good!

Just one string change

**Review by alexjba — APPROVED** [review]
LGTM!

File: ui/imports/shared/stores/CommunityTokensStore.qml [hunk 2]
```diff
@@ -42,6 +42,14 @@ QtObject {
         console.log("TODO: Deploy Asset backend!")
     }
 
+    function deleteToken(communityId, contractUniqueKey) {
+        console.log("TODO: Delete token bakend!")
+    }
+
+    function retryMintToken(communityId, contractUniqueKey) {
```
Comment (endulab) [inline ui/imports/shared/stores/CommunityTokensStore.qml:49]: Is tokenKey the same as contractUniqueKey?

**Review by endulab — APPROVED** [review]

**Review by micieslak — APPROVED** [review]
LGTM!

File: ui/imports/shared/stores/CommunityTokensStore.qml (hunk 2 above)
Comment (noeliaSD) [reply ui/imports/shared/stores/CommunityTokensStore.qml:49]: Yes, it will be renamed in my next pr to be always `tokenKey`

**Pull request merged**